.appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: '1.2.0-{build}'
+clone_depth: 1
+test: off
+environment:
+    global:
+        CYG_ROOT: C:\cygwin
+        CYG_MIRROR: http://cygwin.mirror.constant.com
+        CYG_CACHE: C:\cygwin\var\cache\setup
+        CYG_BASH: C:\cygwin\bin\bash
+
+cache:
+    - '%CYG_CACHE%'
+
+init:
+    - git config --global core.autocrlf input
+
+install:
+    - ps: 'Start-FileDownload "http://cygwin.com/setup-x86.exe" -FileName "setup-x86.exe"'
+    - 'setup-x86.exe --quiet-mode --no-shortcuts --only-site --root "%CYG_ROOT%" --site "%CYG_MIRROR%" --local-package-dir "%CYG_CACHE%" --packages gcc-g++,autoconf,automake,make,cmake,libboost-devel,openssl-devel,bzip2,python,git,libstdc++6-devel,libkrb5-devel > NUL 2>&1'
```

Comment:
I shortened the line a bit by using the environment variable strategy you suggested.